content/pages/sprint/2025/02-hsinchu.rst

```diff
@@ -56,6 +56,22 @@ development insights in this event.
 We would like to provide a supportive and friendly environment for all attendees to support more developers
 to join in the open-source communities. 
 
+Technical sharing
+-------------------
+
+scientific programming to develop code of the space-time conservation element and solution element (CESE) method in Python and C++
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Venue: At the largest room in NTHU CTC**
```

Comment:
I have mentioned that technical sharing will use the largest room in NTHU CTC.

Reply:
LGTM.  Maybe you can just say "in CTC".

Reply:
Sounds good, I will use CTC next time!